src/components/Tool/NavToolList/NavToolList.tsx

```diff
@@ -1,61 +1,153 @@
 import React, { Fragment, ReactElement } from 'react';
 import Icon from '@mdi/react';
+import Close from '@mui/icons-material/Close';
 import {
   Box,
+  IconButton,
   List,
   ListItem,
   ListItemIcon,
   ListItemText,
+  Slide,
   Theme,
+  Typography,
 } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
 import { makeStyles } from 'tss-react/mui';
+import { useGetToolNotificationsQuery } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/GetToolNotifcations.generated';
+import { ToolName } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu';
+import { useAccountListId } from 'src/hooks/useAccountListId';
 import { ToolsList } from '../Home/ToolList';
 import { Item } from './Item/Item';
 
 const useStyles = makeStyles()((theme: Theme) => ({
-  list: {
-    width: '290px',
-    transform: 'translateY(55px)',
-    [theme.breakpoints.down('xs')]: {
-      transform: 'translateY(45px)',
-    },
+  root: {
+    overflow: 'hidden',
   },
   li: {
-    borderTop: '1px solid',
-    borderBottom: '1px solid',
-    borderColor: theme.palette.cruGrayDark.main,
+    borderTop: `1px solid ${theme.palette.cruGrayLight.main}`,
+    borderBottom: `1px solid ${theme.palette.cruGrayLight.main}`,
+    borderColor: theme.palette.cruGrayLight.main,
+  },
+  notificationBox: {
+    backgroundColor: theme.palette.progressBarYellow.main,
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(1),
+    borderRadius: '25%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginLeft: theme.spacing(2),
+  },
+}));
+
+const FilterHeader = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(2),
+  borderBottomColor: theme.palette.grey[200],
+}));
+
+const FilterList = styled(List)(({ theme }) => ({
+  paddingTop: 0,
+  '& .MuiListItemIcon-root': {
+    minWidth: '37px',
+  },
+  '& .FilterListItemMultiselect-root': {
+    marginBottom: theme.spacing(2),
   },
 }));
 
 export interface Props {
   selectedId?: string;
+  isOpen?: boolean;
+  toggle: (isOpen: boolean) => void;
 }
 
-const NavToolList = ({ selectedId }: Props): ReactElement => {
+const NavToolList = ({ selectedId, isOpen, toggle }: Props): ReactElement => {
   const { classes } = useStyles();
+  const { t } = useTranslation();
+  const accountListId = useAccountListId();
+  const { data, loading } = useGetToolNotificationsQuery({
+    variables: { accountListId: accountListId ?? '' },
+    skip: !accountListId,
+  });
+
+  const toolData: { [key: string]: { totalCount: number } } = {
+    [ToolName.FixCommitmentInfo]: data?.[ToolName.FixCommitmentInfo] ?? {
```

Comment:
For all of these lookups, I would just use the GraphQL field name instead of the enum.

```suggestion
    [ToolName.FixCommitmentInfo]: data?.fixCommitmentInfo ?? {
```